content/docs/start/data-pipelines.md

```diff
@@ -1,13 +1,23 @@
+---
+description: |
+  Learn how DVC data pipelines can help you organize, version, and efficiently reproduce your data science or machine learning project.
```

Comment:
We haven't found a good way to automate this, but let's still wrap this at 80 please

Reply:
Thanks for catching this. Done, and I'll apply going forward.